tests/unittests/CoreText/CTFontTests.mm

```diff
@@ -73,8 +72,7 @@ virtual void TearDown() {
 
 INSTANTIATE_TEST_CASE_P(CTFont,
                         FontCopyName,
-                        ::testing::Values(::testing::make_tuple(kCTFontCopyrightNameKey, c_copyrightName),
```

Comment:
(if it's that WinObjC.ttf currently lacks these other metadata, is there a way we could add them to WinObjC.ttf?)

---
In reply to: [97684882](https://github.com/Microsoft/WinObjC/pull/1799#discussion_r97684882) [](ancestors = 97684882)